hw/top_earlgrey/ip/ast/data/ast.hjson

```diff
@@ -44,6 +44,8 @@
         cname: "REGA",
         swaccess: "rw",
         hwaccess: "hro",
+        tags: [ // don't write random data to any of the AST registers
+          "excl:CsrAllTests:CsrExclWrite" ],
```

Comment:
nit: please fix alignment issue here and below:
```suggestion
        tags: [ // don't write random data to any of the AST registers
               "excl:CsrAllTests:CsrExclWrite" ],
```

Reply:
Updated!